aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1492,6 +1517,152 @@ def get_openapi_json_schema(
+        summary: Optional[str] = None,
+        description: Optional[str] = None,
+        tags: Optional[List[str]] = None,
+        servers: Optional[List["Server"]] = None,
+        terms_of_service: Optional[str] = None,
+        contact: Optional["Contact"] = None,
+        license_info: Optional["License"] = None,
+        swagger_base_url: Optional[str] = None,
+        middlewares: Optional[List[Callable[..., Response]]] = None,
+    ):
+        """
+        Returns the OpenAPI schema as a JSON serializable dict
+
+        Parameters
+        ----------
+        path: str, default = "/swagger"
+            The path to the swagger UI.
+        title: str
+            The title of the application.
+        version: str
+            The version of the OpenAPI document (which is distinct from the OpenAPI Specification version or the API
+        openapi_version: str, default = "3.1.0"
+            The version of the OpenAPI Specification (which the document uses).
+        summary: str, optional
+            A short summary of what the application does.
+        description: str, optional
+            A verbose explanation of the application behavior.
+        tags: List[str], optional
+            A list of tags used by the specification with additional metadata.
+        servers: List[Server], optional
+            An array of Server Objects, which provide connectivity information to a target server.
+        terms_of_service: str, optional
+            A URL to the Terms of Service for the API. MUST be in the format of a URL.
+        contact: Contact, optional
+            The contact information for the exposed API.
+        license_info: License, optional
+            The license information for the exposed API.
+        swagger_base_url: str, optional
+            The base url for the swagger UI. If not provided, we will serve a recent version of the Swagger UI.
+        middlewares: List[Callable[..., Response]], optional
+            List of middlewares to be used for the swagger route.
+        """
+        from aws_lambda_powertools.event_handler.openapi.models import Server
+
+        if not swagger_base_url:
+
+            @self.get("/swagger.js", include_in_schema=False)
+            def swagger_js():
+                body = Path.open(Path(__file__).parent / "openapi" / "swagger_ui" / "swagger-ui-bundle.min.js").read()
+                return Response(
+                    status_code=200,
+                    content_type="text/javascript",
+                    body=body,
+                    compress=True,
+                )
+
+            @self.get("/swagger.css", include_in_schema=False)
+            def swagger_css():
+                body = Path.open(Path(__file__).parent / "openapi" / "swagger_ui" / "swagger-ui.min.css").read()
+                return Response(
+                    status_code=200,
+                    content_type="text/css",
+                    body=body,
+                    compress=True,
```

Comment:
I think we need to add a parameter if the customer wants to compress or not. Forcing this we can have issues with net :: ERR_CONTENT_DECODING_FAILED 200 (OK) when running locally.